modules/oxide/src/less/theme/components/toolbar-button/toolbar-split-button.less

```diff
@@ -25,6 +26,7 @@
     z-index: 1; // Ensure focus outline is on top of other buttons
 
     &::after {
+      pointer-events: none; // Prevent the focus outline from opening the menu
```

Comment:
Can this be added in the mix in even or is that strange. Could other things be broken that use this mixin/pseudo element?

Reply:
No I would not dare add that to the mixin.